src/vs/base/common/winjs.base.d.ts

```diff
@@ -134,5 +134,5 @@ export declare class PPromise<C, P> extends TPromise<C> {
 	public static join<C, P>(promises: PPromise<C, P>[]): PPromise<C, P[]>;
 	public static join<C, P>(promises: { [n: string]: PPromise<C, P> }): PPromise<{ [n: string]: C }, P>;
 	public static any<C, P>(promises: PPromise<C, P>[]): PPromise<{ key: string; value: PPromise<C, P>; }, P>;
-	public static wrapError<V>(error: any): TPromise<V>;
+	public static wrapError(error: Error): TPromise<never>;
```

Comment:
Removing the generic type is the cause for all those compile errors. While it seems correct, I'd go the way [`lib.d.ts`](https://github.com/Microsoft/TypeScript/blob/master/lib/lib.es2015.promise.d.ts#L207) did and have both signatures. Especially, when returning a rejected-promise from a function that returns `Promise<T>` isn't possible with the `never`-approch.